editor/src/core/shared/template-path.ts

```diff
@@ -166,7 +167,7 @@ function newScenePath(elementPaths: StaticElementPath[]): ScenePath {
   }
 }
 
-export const emptyScenePath: ScenePath = newScenePath([])
+export const emptyScenePath: StaticScenePath = newScenePath([]) as StaticScenePath
```

Comment:
Isn't this cast dangerous? If the definition of `newScenePath` changes for instance?

Reply:
Fair point. This was based on the `emptyInstancePath` definition, but I'll hand roll it for both.

Reply:
Done